Decompiler/Program.cs

```diff
@@ -115,6 +129,11 @@ private static void ProcessFile(string path)
             {
                 ParseVPK(path);
 
+                if (File.Exists(path + ".manifest.txt"))
+                {
+                    File.Delete(path + ".manifest.txt");
```

Comment:
The manifest.txt is a great addition, I only removed it temporary to force a dump and forgot about it - blaming it on coding Valve-style...  in a text editor :)